server.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const helmet = require("helmet");
+const bcrypt = require("bcryptjs");
+const session = require("express-session");
+const cookieParser = require("cookie-parser");
+const db = require("./data/helpers/users.js");
+
+const server = express();
+server.use(helmet(), cookieParser(), express.json());
+
+server.use(
+  session({
+    name: "notsession",
+    secret: "why is there a secret?",
```

Comment:
Good question! Typically, this secret would be stored in a `.env` file (and/or as a config var via a deployment platform). If you're interested in getting it set up, all of the information you need is [here](https://www.npmjs.com/package/dotenv). Best practice is to put your `.env` in your `.gitignore`.